webapp/src/shared/components/ConfirmAlert/CustomConfirmAlert.css

```diff
@@ -0,0 +1,52 @@
+body.react-confirm-alert-body-element {
+  overflow: hidden;
+}
+
+.react-confirm-alert-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 99;
+  background: var(--color-background);
```

Comment:
I think I like it more without the background, overlapping the current content instead of showing the alert over a black background

Reply:
What amount of background would we like to see? It's as simple as changing the transparency channel of the background attribute. I.e.:
```diff
Index: webapp/src/shared/components/ConfirmAlert/CustomConfirmAlert.css
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/webapp/src/shared/components/ConfirmAlert/CustomConfirmAlert.css b/webapp/src/shared/components/ConfirmAlert/CustomConfirmAlert.css
--- a/webapp/src/shared/components/ConfirmAlert/CustomConfirmAlert.css	(revision 81007dfe565f77de2f21f7e4121d8cacba2718ab)
+++ b/webapp/src/shared/components/ConfirmAlert/CustomConfirmAlert.css	(date 1674736988733)
@@ -9,7 +9,7 @@
   right: 0;
   bottom: 0;
   z-index: 99;
-  background: var(--color-background);
+  background: rgb(0, 0, 0, 0.8);
   display: flex;
   justify-content: center;
   align-items: center;
```